lib/objects/sensor.ts

```
@@ -0,0 +1,181 @@
+  }
+
+  /**
+   * Find the closest intersection point between `ray` and any polygon in
+   * `traffic`.
+   *
+   * @param ray - Ray segment to test
+   * @param traffic - Array of other cars whose polygons will be tested
+   * @returns The nearest `Intersection` along the ray, or `null` if none
+   */
+  private getReading(ray: Edge, traffic: Car[]) {
+    let touches = [];
```

ESLint

[error] 'touches' is never reassigned. Use 'const' instead.

(prefer-const)